tfhe/src/integer/server_key/radix_parallel/tests_unsigned/test_scalar_comparison.rs

```diff
@@ -398,6 +398,38 @@ fn integer_unchecked_scalar_comparisons_edge(param: ClassicPBSParameters) {
     }
 }
 
+// Tests whether equality holds for greater/less or equal than, but fails for greater/lesser than.
```

Comment:
This comment is no longer really valid

Reply:
I updated the comment. Let me know if you want it removed entirely